### DISCUSSION TITLE
CRD's not working with Kubeconform

### DISCUSSION THREAD

**Discussion by dazedavid** [kedacore/charts#208] [Q&A]
### Use-Case
Trying to use the kubeconform to validate the charts before using it getting following errors:
`helm template ./keda | kubeconform -output json`

{
  "resources": [
    {
      "filename": "stdin",
      "kind": "CustomResourceDefinition",
      "name": "scaledjobs.keda.sh",
      "version": "apiextensions.k8s.io/v1",
      "status": "statusError",
      "msg": "could not find schema for CustomResourceDefinition"
    },
    {
      "filename": "stdin",
      "kind": "CustomResourceDefinition",
      "name": "triggerauthentications.keda.sh",
      "version": "apiextensions.k8s.io/v1",
      "status": "statusError",
      "msg": "could not find schema for CustomResourceDefinition"
    },
    {
      "filename": "stdin",
      "kind": "CustomResourceDefinition",
      "name": "scaledobjects.keda.sh",
      "version": "apiextensions.k8s.io/v1",
      "status": "statusError",
      "msg": "could not find schema for CustomResourceDefinition"
    },
    {
      "filename": "stdin",
      "kind": "CustomResourceDefinition",
      "name": "clustertriggerauthentications.keda.sh",
      "version": "apiextensions.k8s.io/v1",
      "status": "statusError",
      "msg": "could not find schema for CustomResourceDefinition"
    }
  ]
}

Tell us more what you'd like to achieve

### Specification

- [ x ] Can anyone help with the issue.

**Comment by tomkerkhove** [comment]
They have been updated, can you please try the latest version?
